apps/web/src/core/components/hooks/useWindowAI.ts

```diff
@@ -18,9 +16,11 @@ export const initialMessages: ChatMessage[] = [
 
 export function useWindowAI(
   defaultMessages = initialMessages,
-  { cacheSize = 10, stream = false } = {}
+  { cacheSize = 10, stream = false, keep = 0, maxTokens = 2048 } = {}
```

Comment:
maybe `keep` should be called `prefixMessageCount` or something, since it's always at the beginning? or maybe I'm confused about the difference between cacheSize and keep

Reply:
Good call - let's do that. And yeah "keep" persist the first few messages in the current cache, mainly to retain the original system message.